content-test/selectors/selectors.test.js

```diff
@@ -117,5 +111,24 @@ describe("selectors", () => {
       const groups = [state.TopSites.rows, state.Spotlight.rows, state.TopActivity.rows];
       assert.deepEqual(groups, dedupe.group(groups));
     });
+    it("should sort TopActivity by dateLastVisited", () => {
+      state = selectNewTabSites({
+        TopSites: {rows: []},
+        Spotlight: {rows: []},
+        FrecentHistory: {rows: [
+          {url: "foo.com", lastVisitDate: 1},
+          {url: "bar.com", lastVisitDate: 4},
+          {url: "baz.com", lastVisitDate: 3}
+        ]}
+      });
+      console.log(state.TopActivity.rows);
```

Comment:
Yeah, I can do a quick PR tomorrow to hide the `no-warning-comments` (TODO) rule, and maybe hide the `no-shadow` rule for now too. That would put us at 0 warnings, making anything new fairly visible (or as visible as warnings can be).